### PR TITLE
Third api

### PR DESCRIPTION
**Pull request by serhiy2000**
Added repo tests.
Added Third API (google places).

### CODE REVIEW

File: src/main/java/lits/jp/hotel/management/places/response.txt [hunk 1]
```diff
@@ -0,0 +1 @@
...�����������, 7, ����"}, {"geometry":{"location":{"lat":49.841673,"lng":24.030309 },"viewport":{"northeast":{"lat":49.8430596802915,"lng":24.0318581302915 },"southwest":{"lat":49.8403617197085,"lng":24.02916016970849 } } },"icon":"https://maps.gstatic.com/mapfiles/place_api/icons/cafe-71.png","id":"8167e175335a35e920357b2480cea050baa5dc51","name":"���� ����","opening_hours":{"open_now":false },"photos":[ {"height":1251,"html_attributions":["\u003ca href=\"https://maps.google.com/maps/contrib/116420520608877579677/photos\"\u003e���� ����\u003c/a\u003e"],"photo_reference":"CmRaAAAA8gdV5Z15-jTvdA8ncv8ZcjX9z7wnV5q7hVFbvab8DUZ3e6FMjLmLxXN3bWgWgSRf4R8A5vYTszVEVQi-aoAfbyU1J6SPBYRbzcrB6kTma4-GZtVFoRzZvcQPI6h8cRMJEhDDK0sXPLq8GDAR5q0RURhhGhSjsQDvjc-vZ9-m19uGcN5Zi1WdHw","width":1875 } ],"place_id":"ChIJs--2vW3dOkcRQpB3AkElOg0","plus_code":{"compound_code":"R2RJ+M4 ����, �������� �������, ������","global_code":"8GX6R2RJ+M4"},"price_level":2,"rating":4.4,"reference":"ChIJs--2vW3dOkcRQpB3AkElOg0","scope":"GOOGLE","placeTypes":["cafe","night_club","bar","food","point_of_interest","establishment"],"user_ratings_total":432,"vicinity":"���'����� �������������, ����� �����, 29, ����"} ],"status":"OK"}
```
Comment (pprotas-j) [inline src/main/java/lits/jp/hotel/management/places/response.txt:1]: Is this a configuration file or a generated response? If the latter, can it not be on the repo?

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,8 +1,8 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/hotel?useSSL=false&characterEncoding=UTF-8&zeroDateTimeBehavior=convertToNull&serverTimezone=UTC
```
Comment (pprotas-j) [inline src/main/resources/application.properties:1]: Add file to .gitignore, please.

**Review by RybakY — APPROVED** [review]

File: src/main/java/lits/jp/hotel/management/places/response.txt (hunk 1 above)
Comment (serhiy2000) [reply src/main/java/lits/jp/hotel/management/places/response.txt:1]: It is generated response. there is a limit for real responses per day from Google.

File: src/main/resources/application.properties (hunk 2 above)
Comment (serhiy2000) [reply src/main/resources/application.properties:1]: It is in my project already. But seems not to be working.

Comment (pprotas-j) [reply src/main/resources/application.properties:1]: The current .gitignore file is the one that's generated, to ignore IDE specific files. We can add the following ```./src/main/resources/application.properties``` to ignore that specific file

**Review by pprotas-j — APPROVED** [review]

**Pull request merged**